hazelcast/src/main/java/com/hazelcast/map/impl/operation/ClearOperation.java

```diff
@@ -41,10 +41,6 @@ public ClearOperation(String name) {
 
     @Override
     public void run() {
-        // near-cache clear will be called multiple times by each clear operation,
```

Comment:
Should be tested via `NearCacheTest.testMapClear_clears_localNearCache`.

:+1: